[jobs] Add support for redis queue jobs

jjmerchante opened the pull request:
This PR adds support for redis queues.
It includes a function to get a job by its ID and another to get the list of the available jobs in the registry.

I couldn't test get_jobs() method using fakeredis but I tested it with a redis server and a worker and it works.

Related to https://github.com/chaoss/grimoirelab-bestiary/issues/87

jjmerchante committed:
[jobs] Add support for redis queue jobs

This commits adds support for redis queues and
adds a function to get a job by its ID and another
to get the list of the available jobs in the registry.

Signed-off-by: JJMerchante <jj.merchante@gmail.com>

evamillan commented:
The code looks good to me, but I'm missing the GraphQL resolvers to get the list of jobs or find a job, will that be on a different PR?

jjmerchante committed:
[jobs] GraphQL resolvers for redis queue jobs

This commits adds the necessary methods for listing and
retrieving jobs using GraphQL.

Signed-off-by: JJMerchante <jj.merchante@gmail.com>

evamillan reviewed (approved):
LGTM
